src/creeps/Zombie.cpp

```diff
@@ -153,72 +276,87 @@ void Zombie::generateMove() {
 /**
  * A* algo generates a string of direction digits.
  * Fred Yang
+ * March 15
+ */
+string Zombie::generatePath(const Point& start) {
+    return generatePath(start, Point(MAP_WIDTH/2, MAP_HEIGHT/2));
+}
```

Comment:
Is this function necessary if it's just going to call essentially the same function?